docs/releases/5.1.txt

```diff
@@ -190,7 +190,8 @@ Cache
 CSRF
 ~~~~
 
-* ...
+* The error messaging is improved when ``Origin`` checking fails,
+  including better hints when likely proxy headers are detected.
```

Comment:
```suggestion
* The ``'403_csrf.html'`` template now has improved messaging when ``Origin`` 
  checking fails, including better hints when likely proxy headers are detected.
```
[Docs are wrapped at 80 characters](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files) and we will now need to rebase and move this to the 5.2 release notes.